.github/workflows/pull-request-labeler.yml

```diff
@@ -1,16 +1,17 @@
 name: "Pull Request Labeler"
 on:
   pull_request_target:
-    types: [created]
+    types: [opened]
 
 jobs:
 
   label_pull_request:
     runs-on: ubuntu-latest
     steps:
-    - uses: thomasjpfan/labeler@v2.5.0
+    - uses: actions/labeler@v4.0.1
```

Comment:
Please test this on your fork. Last time we had this action, the sync mechanism was not working. The issue is still open with a workaround I did not test https://github.com/actions/labeler/pull/113

Reply:
I am comming back to this PR after an eternity: I plan to maintain the changes of 
https://github.com/actions/labeler/pull/113 on my fork in the meantime. I still need to assert whether it works or not. If it does, would depending on this fork work for you or would you prefer depending on https://github.com/actions/labeler/ solely?